assignment.js

```diff
@@ -32,6 +32,12 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const baseNumber = 2;
```

Comment:
@annayensietopping great job using a for loop here, code was well structured and easy to follow. An example of an alternative approach to this problem using a while loop [can be found here](https://github.com/jsd20200127/hw-01-js-basics-control-flow-loops-solution/blob/master/assignment.js#L38)